Release Zulip Server 7.0.

timabbott opened the pull request.

andersk commented on docs/overview/changelog.md:
```diff
@@ -5,11 +5,12 @@ This page the release history for the Zulip server. See also the
-This section is an incomplete draft of the release notes for the next
-major release, and is only updated occasionally. See the [commit
-log][commit-log] for an up-to-date list of all changes.
+This page contains the release history for the Zulip 6.x stable
```
7.x?

andersk commented on docs/overview/changelog.md:
```diff
@@ -5,11 +5,12 @@ This page the release history for the Zulip server. See also the
-This section is an incomplete draft of the release notes for the next
-major release, and is only updated occasionally. See the [commit
-log][commit-log] for an up-to-date list of all changes.
+This page contains the release history for the Zulip 6.x stable
+release series. See the [current Zulip changelog][latest-changelog]
+for newer release series, or the [commit log][commit-log] for an
+up-to-date list of all changes.
```
This paragraph seems like it belongs above under `# Version history`, not here under `### 7.0 -- 2023-05-31`. That’s what we did for 6.0.

timabbott commented:
Thanks for catching those! Fixed both of those issues, and also dropped the link to the release blog update, since the blog post will be posted after this is merged.

timabbott commented:
Merged as b00ce1bad15af7a62d333634d5295962c8a2aa69 via the release command.
